lib/codeowners/cli/main.rb

```diff
@@ -77,10 +77,13 @@ def create_checker(repo)
       }.freeze
 
       def report_errors!(checker)
-        checker.fix!.each do |error_type, inconsistencies|
-          next if inconsistencies.empty?
-
-          puts LABELS[error_type], '-' * 30, inconsistencies, '-' * 30
+        checker.fix!.reduce(nil) do |prev_error_type, (error_type, inconsistencies, meta)|
+          puts('-' * 30, LABELS[error_type], '-' * 30) if prev_error_type != error_type
```

Comment:
Could we extract that into `print_delimiter_line(type)` or something like that? It would make the block more readable.